Formula/icecast.rb

```diff
@@ -23,4 +26,19 @@ def install
     (prefix+"var/log/icecast").mkpath
     touch prefix+"var/log/icecast/error.log"
   end
+
+  test do
+    ICECAST_PID = spawn "icecast", "-c", "#{prefix}/etc/icecast.xml", "2>", "/dev/null"
+    sleep(3)
+
+    begin
+      system "kill", "-0", ICECAST_PID
+      assert_equal 0, $CHILD_STATUS.exitstatus
```

Comment:
Please search `$(brew --repository)/Library/Taps/homebrew/homebrew-core/Formula` for either `CHILD_STATUS` or `pid`. You should not need to `require "English"` for this to work. If you must `require` something, please use [`Open3`](https://docs.ruby-lang.org/en/2.0.0/Open3.html) and put the `require` for the test inside of the `test do` block. You can look at https://github.com/Homebrew/homebrew-core/pull/38752/files or any one of the formula that come up when searching for `pid` or `CHILD_STATUS` for inspiration and guidance.

Thanks!